handshake_manager_test.go

```diff
@@ -71,7 +71,7 @@ func Test_NewHandshakeManagerTrigger(t *testing.T) {
 
 	assert.Equal(t, 0, testCountTimerWheelEntries(blah.OutboundHandshakeTimer))
 
-	hi := blah.AddVpnIp(ip)
+	hi := blah.AddVpnIp(ip, nil)
```

Comment:
Lets add a test case to ensure our init func is called and the created return bool is correct

Reply:
Test added higher up in the file in latest commit